### ISSUE TITLE
[BUG] Unable to create user with Magpie 3.8.0

### ISSUE THREAD

**Issue by tlvu** [Ouranosinc/Magpie#427]
**Describe the bug**
Creating a new user (username: lvu2, email: a@b.c, password: more_than_12_char, group: users) result in this error in the logs (`docker logs magpie`):

```
[2021-05-04 13:37:17,017] INFO       [ThreadPoolExecutor-0_0][magpie.api.requests] User 'admin' is authenticated
[2021-05-04 13:37:17,017] INFO       [ThreadPoolExecutor-0_0][magpie.api.requests] User 'admin' is authenticated
[2021-05-04 13:37:17,025] ERROR      [ThreadPoolExecutor-0_0][magpie.ui.utils] Unexpected API error under UI operation. [TypeError: argument of type 'Response' is not iterable]
Traceback (most recent call last):
  File "/opt/local/src/magpie/magpie/ui/utils.py", line 123, in wrap
    return func(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/pyramid/viewderivers.py", line 436, in rendered_view
    result = view(context, request)
  File "/usr/local/lib/python3.7/site-packages/pyramid/viewderivers.py", line 116, in _class_requestonly_view
    response = getattr(inst, attr)()
  File "/opt/local/src/magpie/magpie/ui/management/views.py", line 232, in add_user
    if user_email in self.get_user_emails():
TypeError: argument of type 'Response' is not iterable
[2021-05-04 13:37:17,034] INFO       [ThreadPoolExecutor-0_0][magpie.utils] Request: [POST pavics.ouranos.ca:443 /magpie/ui/error]
```

![Screenshot_2021-05-04 Magpie](https://user-images.githubusercontent.com/11966697/117012406-e7ef9780-acbc-11eb-82e7-cb30283dbe2f.png)

**Desktop (please complete the following information):**
 - OS: Linux
 - Browser: Firefox 
 - Instance: https://pavics.ouranos.ca/magpie/ui/users/add
 - Version: 3.8.0 (from birdhouse-deploy stack)

**Impact**

This is blocking us from creating new Jupyter users in production.  If a work-around exists without waiting for a new version with a fix, please let us know.

@tlogan2000 FYI.

**Comment by tlvu** [comment]
Found a work-around, using https://github.com/bird-house/birdhouse-deploy/blob/02d1e9db0cfc0ca0c2ca382e379228a48d4ca182/birdhouse/scripts/create-magpie-authtest-user#L13-L19 and replacing that config with the user we want to create.

**Comment by fmigneault** [comment]
@tlvu 
As a workaround, you can do the calls via API or CLI until the fix is integrated. Seems the issue is only specific to the UI.

**Comment by fmigneault** [comment]
fix available in 3.10.1
